### DISCUSSION TITLE
Calendar usage - WorkDays

### DISCUSSION THREAD

**Discussion by albarmat** [apache/dolphinscheduler#13056] [Ideas]
I am starting using Dolphin Scheduler for essentially executing some shell command lines.
I don't know yet how to schedule tasks at a specific dates typically :
- run the task only on the first day of the month
- run from monday to tuesday.
- define the holidays and don't run the jobs on those days

 Is there a sort of calendar that can be plugged to my tasks / workflows ? 

thanks

**Comment by DarkAssassinator** [comment]
hi @albarmat ,
+ run the task only on the first day of the month  -> please check the `cron`  setting
+ run from monday to tuesday. -> please check the `cron`  setting
+ define the holidays and don't run the jobs on those days -> as i know, current DS do not support, but i think that this is a good idea.
